docs/sql/reference/refresh.txt

```diff
@@ -5,19 +5,19 @@
 REFRESH
 =======
 
-Refresh a table explicitly.
+Refresh a table or multiple tables explicitly.
 
 Synopsis
 ========
 
 ::
 
-    REFRESH TABLE table_ident [ PARTITION (partition_column=value [ , ... ]) ]
+    REFRESH TABLE [ table_ident PARTITION (partition_column=value [ , ... ]) ] [ , ...]
```

Comment:
I think that `REFRESH TABLE table_ident PARTITION (partition_column=value [ , ... ]) [ , ...]` is correct, since `[ ]` means optional